### PR TITLE
Grammer fix in snapshot doc "haven't" -> "hasn't"

### PR DESCRIPTION
**Pull request by ChillerDragon**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/engine/docs/snapshots.txt [hunk 1]
```diff
@@ -54,4 +54,4 @@ The interval for how often a client receives a snapshot changes during the cours
 
 - *Full*. Snapshot for every tick or every even tick depending on server configuration. This is used during normal gameplay and everything is running smooth.
 
-- *Recovery*. 1 snapshot each second. A client enters recovery rate when it haven't acknowlaged a snapshot over 1 second. This is to let the client to beable to recover if it has a slow connection.
+- *Recovery*. 1 snapshot each second. A client enters recovery rate when it hasn't acknowlaged a snapshot over 1 second. This is to let the client to beable to recover if it has a slow connection.
```
Comment (def-) [inline src/engine/docs/snapshots.txt:57]: acknowledged*
to beable -> be able

**Review by def- — CHANGES_REQUESTED** [review]

File: src/engine/docs/snapshots.txt (hunk 1 above)
Comment (ChillerDragon) [reply src/engine/docs/snapshots.txt:57]: here we go again :D

**Comment by ChillerDragon** [comment]
This doc is so cursed haha

... but its all we got o.O

**Comment by def-** [comment]
bors r+
Still weird grammar, but better than before.

**Comment by ChillerDragon** [comment]
> bors r+ Still weird grammar, but better than before.

agree

**Pull request merged**